forge-gui/res/cardsfolder/upcoming/escape_detection.txt

```diff
@@ -0,0 +1,7 @@
+Name:Escape Detection
+ManaCost:1 U U
+Types:Instant
+K:Freerunning:Return<1/Creature.Blue>
```

Comment:
@Hanmac Don't think this is wired up yet ?

Reply:
the Cost should work, just not the formatting

@tool4ever you can look into this too

Reply:
I think I'll remove this one for now